tests/gtest/fake_platform.cpp

```diff
@@ -577,4 +601,8 @@ otError otPlatUdpLeaveMulticastGroup(otUdpSocket *, otNetifIdentifier, const otI
     return OT_ERROR_NOT_IMPLEMENTED;
 }
 void otPlatAssertFail(const char *, int) {}
+
+otError otPlatRadioEnableCsl(otInstance *, uint32_t, otShortAddress, const otExtAddress *) { return OT_ERROR_NONE; }
+void    otPlatRadioUpdateCslSampleTime(otInstance *, uint32_t) {}
```

Comment:
Just want to make sure if these are added intentionally.

Reply:
should be in a separate PR.